packages/relay-runtime/util/withDuration.js

```diff
@@ -13,7 +13,7 @@
 'use strict';
 
 const isPerformanceNowAvailable =
-  global.performance != null && typeof global.performance.now === 'function';
+  global?.performance != null && typeof global.performance.now === 'function';
```

Comment:
It might be safer to check for existence like this:

```js
const isPerformanceNowAvailable = 
  typeof window !== 'undefined' && 
  typeof window?.performance?.now === 'function';
```

1. `whatever?.foo` where `wherever` is not a defined identifier will throw an uncaught ReferenceError
2. `window` is more appropriate here since we're checking for a browser only API. Only node.js defines `global` AFAIK.

Reply:
@poteto great input, I'll update the PR